hooks/event-conversion-utils.ts

```diff
@@ -0,0 +1,125 @@
+import Airtable from 'airtable'
+import { CommunityEvent, CommunityEventType, WorldLocation } from '~/store/modules/events.ts'
+
+export {
+  fetchCommunityEvents,
+  convertToCommunityEvent,
+  getName,
+  getType,
+  getImage,
+  getPlace,
+  getLocation,
+  getWebsite,
+  getDates,
+  formatDates
+}
+
+async function fetchCommunityEvents (apiKey: string, { days }): Promise<CommunityEvent[]> {
+  const events: CommunityEvent[] = []
+  const base = new Airtable({ apiKey }).base('appkaaRF2QdwfusP1')
+  await base('Events Master View').select({
+    fields: [
+      'Name',
+      'Start Date',
+      'End Date',
+      'Type of Event',
+      'Event Website',
+      'Event Location'
+    ],
+    filterByFormula: `AND(
+      DATETIME_DIFF({Start Date}, TODAY(), 'days') ${days > 0 ? '<=' : '>='} ${days},
+      DATETIME_DIFF({Start Date}, TODAY(), 'days') ${days > 0 ? '>=' : '<'} 0,
+      FIND("IBMers Attending", {What do we send? (Involvement)}) > 0
+    )`,
+    sort: [{ field: 'Start Date', direction: days > 0 ? 'asc' : 'desc' }]
+  }).eachPage((records: any[], nextPage: () => any) => {
+    for (const record of records) {
+      const event = convertToCommunityEvent(record)
```

Comment:
Same as before: In order to be consistent with the rest of the code, I think this variable should be renamed to `communityEvent`